test/setup/setup.ts

```diff
@@ -130,6 +130,7 @@ beforeAll(async () => {
 });
 
 const globalGitHubAppConfig = {
+	gitHubAppId: -1, //TODO: ARC-1619, this value is not known yet.
```

Comment:
shouldn't this be `undefined`?

Reply:
I've removed `globalGitHubAppConfig` completely, it is not being used at all. And from the value itself, I can't tell whether it is a cloud config or a ghe config. So remove it.